index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<title>My Portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <img class="nav-image" src="images/image1.png" alt="image1">
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+    <section class="section">
+            <h1>Rudy<br>Menekam</h1>
+        <p> Hello! I am a software developer and designer! I can help you build a product, feature or website. Taka a look at my works.
+            If you like what you see and have a project you need coded or designed, don’t hesitate and contact me.</p>
+
+        <a href="#" class="button">Start collaboration</a>
+        
+        <!-- <div class="section-image"></div> -->
+    </section>
+    <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
+    <div class="section">
```

Comment:
- [ ] I think it is better to remove the `<br><br><br><br><be>.....` to make your code clean :relaxed:

Reply:
hehe I know, but that is the only hack I found for now
but will still do my research to fix it